tasks/cks/labs/19/SOLUTION.MD

```diff
@@ -1,28 +1,40 @@
-```
-# vim  /var/work/14/Dockerfile
-
+# Read docker [build best practies](https://docs.docker.com/build/building/best-practices/)
 
+# Fix dockerfile
+```
+vim  /var/work/14/Dockerfile
+```
+*It should looks like this one:*
+```
 FROM ubuntu:20.04
-RUN apt-get update
-RUN apt-get -y install curl
+RUN apt-get update && apt-get -y install curl
```

Comment:
What's the sense of this fix if it's the same?

Reply:
This isn't the same.
Check [best practices](https://docs.docker.com/build/building/best-practices/#apt-get) please

Reply:
There is such a question in the real exam . and there you only need to add creating a user . and running the app as that user . changing other lines of the dockerfile may cause this question to fail .